settings.gradle

```diff
@@ -23,6 +23,8 @@ include 'eventmesh-common'
 include 'eventmesh-starter'
 include 'eventmesh-examples'
 include 'eventmesh-spi'
+include 'eventmesh-filter'
+include 'eventmesh-transform'
```

Comment:
`filter` is a noun however `transform` is a verb. How about `eventmesh-transformer`?

Reply:
fixed